store/postgres/tag_repository_test.go

```diff
@@ -26,21 +26,21 @@ type TagRepositoryTestSuite struct {
 func (r *TagRepositoryTestSuite) SetupSuite() {
 	var err error
 
-	logger := logrus.New()
+	logger := log.NewLogrus()
```

Comment:
there is `log.NewNoop` from salt to stub logger.

Reply:
this logger was needed to debug with dockertest. But sure we can use Noop too.

Reply:
Oh if it is needed then it's fine 👌 . Was just an fyi.

Reply:
no worries, thanks for pointing out stew, I have just updated it to Noop in PR, we can easily make it `NewLogrus` if we want to debug it though.